src/main/java/com/example/usermanagement/dto/LoginDTO.java

```diff
@@ -4,7 +4,7 @@
 import lombok.Data;
 
 @Data
-public class LoginDto {
+public class LoginDTO {
 
     @NotEmpty(message = "Username cannot be null or empty")
```

Comment:
check if username ="       " work with @NotEmpty

Reply:
It gives me 401 unauthorised which shows Invalid Username or Password.
![Screenshot from 2023-06-19 17-30-24](https://github.com/prakharsaxena3007/user-management/assets/133750194/38198fe0-80ba-42ae-90dd-20a9cbfc8b28)

Reply:
it is giving unauthorized due to wrong username ,but if we use MethodArgumentInvalidException for validating this field with @NotEmpty it will not throw exception so better we can use @NotBlank